### PR TITLE
swarm: creates a 200 server response when localhost is pinged by JSON

### PR DESCRIPTION
**Pull request by cooganb**
Addresses issue #185

### CODE REVIEW

File: swarm/api/http/server.go [hunk 1]
```diff
@@ -591,6 +591,12 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	s.logDebug("HTTP %s request URL: '%s', Host: '%s', Path: '%s', Referer: '%s', Accept: '%s'", r.Method, r.RequestURI, r.URL.Host, r.URL.Path, r.Referer(), r.Header.Get("Accept"))
 
+	if r.RequestURI == "/" && strings.Contains(r.Header.Get("Accept"), "application/json") {
+		w.Header().Set("Content-Type","application/json")
+		w.WriteHeader(http.StatusOK)
+		return
```
Comment (lmars) [inline swarm/api/http/server.go]: It should actually write valid JSON in the response. The simplest thing is to just write a string:

```go
json.NewEncoder(w).Write("welcome to Swarm!")
```

Comment (cooganb) [reply swarm/api/http/server.go]: @lmars Gotcha, replace all the current response code with just `json.NewEncoder`?

Comment (gbalint) [reply swarm/api/http/server.go]: @cooganb The response code can stay, but you have to add JSON content to the response

**Comment by cooganb** [comment]
@gbalint passing all tests except one which is failing due to whisper

**Comment by zelig** [comment]
obsoleted by #412

**Pull request closed without merging**